doc/compiler/runtime/J9JITExceptionTable.md

```diff
@@ -0,0 +1,400 @@
+<!--
+Copyright IBM Corp. and others 2023
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] https://openjdk.org/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+-->
+
+# Background
+
+The `J9JITExceptionTable` (also `typedef`d to `TR_MethodMetaData`)
+is generally referred to as "the metadata". It contains most of the
+metadata associated with a compiled body (the other metadata
+is stored in the `TR_PersistentJittedBodyInfo`). Most of this metadata
```

Comment:
Would it be helpful to mention just a bit about what kind of data are stored in `TR_PersistentJittedBodyInfo` but not in `J9JITExceptionTable`? When I read this sentence, I was wondering why the metadata for the compiled body is stored in two data structures or locations.

Reply:
yeah I can mention some of the data. As for why it's stored in two, probably a historical thing; it's definitely befor emy time, but my guess is that it has to do with the fact that back when the compiler component and the VM component were in different code bases, it was more cumbersome to update the `J9JITExceptionTable` (whose definition existed in the VM component) so the `TR_PersistentJittedBodyInfo` was used instead.